test/inspector/test-inspector-port-cluster.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const assert = require('assert');
+const cluster = require('cluster');
+const common = require('../common');
+const debuggerPort = common.PORT;
+const childProcess = require('child_process');
+
+common.skipIfInspectorDisabled();
+
+let offset = 0;
+
+/*
+ * This test suite checks that inspector port in cluster is incremented
+ * for different execArgv combinations
+ */
+
+function testRunnerMain() {
+  spawnMaster({
+    execArgv: ['--inspect'],
+    workers: [{expectedPort: 9230}]
+  });
+
+  let port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: [`--inspect=${port}`],
+    workers: [
```

Comment:
> Port collisions between tests within single test run?

Yes. Probably will not be a problem if you use `common.PORT` but if you use 0 the OS give other processes ports in that range. We're not sure to whom, maybe Jenkins communication, but we've seen port clashes not rarely.